### DISCUSSION TITLE
Reorganization of the "Systems" category of outfits

### DISCUSSION THREAD

**Discussion by czartrak** [endless-sky/endless-sky#7923] [Ideas]
After some discussion on the discord, it was agreed upon that the current "systems" category is a bit of a mess of a lot of different outfits with very different purposes. My proposed solution is to add several new categories, in order to better organize the outfitter and help you find what you need. My current proposal is as follows:

- Shields/Hull - self explanatory
- Avionics - Scanner/EWar
- Cooling - explanatory
- Accomodations - different rooms/research lab
- Utility - expansions
- Space Travel - Drive/Fuel/Etc.
- Miscellaneous - Systems core/plating/other things that dont fit in a specific category

This will definitely be a lot of monotonous replacing if this is to be PR'd so feedback beforehand is appreciated

**Comment by 1010todd** [comment]
Shields/Hull -> Maintenance? 
Sys Core can probably fit into Shields/Hull | Maintenance. Interference plating might fit in Avionics.
Miscellaneous might be a bit too general as the word could cover something else unrelated to ships, maybe keep it as Systems?

**Reply by czartrak** [reply]
I personally didnt think having Plating under "avionics" made sense from a "realistic" standpoint since the category was for electronic parts, and the Plating is more akin to say, the stealth coating on a jet fighter

**Comment by Zitchas** [comment]
While I agree it'd be nice to break up "Systems" into a few more categories, I think I'd prefer to just split it in half, or thereabouts. Maybe three categories? Most of the time I don't see much problem with its current size. The Systems section in human space might be approaching an unwieldy size, but nowhere else is. The only place where it's really a problem is in the map outfitter list, and I find myself not really using the map outfitter much for anything in systems because there's so few things to actually compare. Weapons, engines, and power systems are really the main things that we have *lots* of and really benefit from comparing, and those are already in their own categories. 

I really don't like the end effect of splitting it into 7 or more categories, which will namely mean making scrolling through the outfits list that much longer, with a lot more empty black space, as this will basically make the outfitter much narrower but taller. 

If I was doing this, I'd probably just pick one or two things to pull out of systems. 

"Ship Integrity Systems" would be one such category: All the shield and hull outfits. Including systems cores.  I think there's enough in here to be worth having it as a category.

As a "problem" with too much differentiation: You mention accommodations and include the Research Lab. But the Research lab is nothing but an asteroid scanner + cargo scanner with a different name that happens to require a crew member. Pulling out rooms as a special category thus only really makes sense if rooms ever become a resource that we use. Ex. If ships could only have a fixed number of additional rooms, perhaps tiny ships couldn't have any, small ships had 1, medium ships only got 2 rooms, and only the largest ships could have several. Then it'd make sense to have rooms grouped together, since something like a Research lab might be directly competing with Luxury quarters for the same "spot." Currently, however, this is not the case. The only thing that the two "compete" for is regular outfitting space.

Alternatively, perhaps just splitting it into "Core Systems" and "Peripheral Systems" would suffice to reduce the size of the category while not making things too much of a "thin column with a lot of wasted space" look.

Core systems being all the shields/hull, cooling, and space travel systems. Peripheral being all the rest (avionics, accommodations, utility, and miscellaneous.)

**Reply by Zireael07** [reply]
+1 to core/peripheral split. 7 categories is too many on reflection and pulling just one or two things and leaving the rest a "grab bag" isn't to appealing either imho

**Reply by czartrak** [reply]
I like the idea of "Ship Integrity Systems" as a category name, and as a way to fit systems cores in. Perhaps Interference Plating and the special rooms could fit under "Utility" to cut off another category? because those really were the only "odd-one-out" outfits (that I can think of at the very least) that would currently inhabit miscellaneous

**Reply by MasterOfGrey** [reply]
To my mind, Core/Peripheral doesn't quite do it, and also doesn't tell me a lot about what's in there, inherently.

I would second a suggestion of **"Ship Integrity Systems"** for anything that adds hull or shield regen, even if they add other things.

Given the proliferation of cooling systems recently, and the fact that you actually *do* want to compare one cooler to another and so might justifiably use the map outfitter to have a look at things - I would also support **"Thermal Management"** (or just "Thermal") as a category.
That also allows for expansions to heat mechanics in the future without locking it into "Cooling".

I would leave everything else in the same grouping though, since they're not generally comparable to each other across multiple sets and they're not particularly numerous in vanilla at any one location.
I would support renaming that group to **"Utility"** though.

**Comment by BaddestApple** [comment]
I'm using a similar configuration with some minor differences, and using a fair amount of custom content & a couple flavor items. Since there's currently no way to order the outfits within categories other than alphabetically, I prefer to have more categories with specific items so they're not only easier to find but also to further sort out items by closing the category folder. For example, I made separate categories for Drives, Fuel (fuel and ramscoops), and Afterburners though they could all work as a single category for vanilla content. Here's a mockup of my current configuration:

<img width="2560" alt="outfit categories" src="https://user-images.githubusercontent.com/79822893/213806549-b086e9f1-cf45-440d-86dc-0b4a4ae7973a.png">

I chose to use Thermal instead of Cooling since some of the items have names with Heat in them (even though they provide cooling), like heat shunts and heat sinks. I also used Tactical instead Avionics but I think Avionics would work well if maybe plating was moved to Shields/Hull; also moved my asteroid scanners to a separate Mining category. Went with Crew and Passengers since I use that category for bunks & all transport mission related outfits but Accommodations probably works better for vanilla. I have expansions in a Retrofits category along with some added illicit ones for weapons and engines and illegal smuggling ones. 

Which actually doesn't leave much for the original Systems category for vanilla content, but however it gets restructured for official release would be better than the mess it is now.

**Reply by Zitchas** [reply]
What plug-ins are you using? There's a lot of outfits I don't recognize there.

**Comment by warp-core** [comment]
Related: #3375
